build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -2000,6 +2188,42 @@ message Compressor {
   }
 }
 
+// Content-defined chunking algorithms used for splitting blobs into chunks.
+message ChunkingAlgorithm {
+  enum Value {
+    // No special constraints about the used chunking algorithm. If a client
+    // selects this value this means he does not care about which exact chunking
+    // algorithm is used. Thus, the server is free to use any chunking algorithm
+    // at its disposal. A server does not need to advertise this value.
+    DEFAULT = 0;
+
+    // Content-defined chunking using Rabin fingerprints. An implementation of
+    // this scheme in presented in this paper
+    // https://link.springer.com/chapter/10.1007/978-1-4613-9323-8_11. The final
+    // implementation of this algorithm should be configured to have the
+    // following properties on resulting chunk sizes.
+    //  - Minimum chunk size: 128 KB
+    //  - Average chunk size: 512 KB (0x000000000007FFFF bit mask)
+    //  - Maximum chunk size: 2048 KB
```

Comment:
Assuming my understanding of the algorithm is correct, doesn't that mean that there is a...
```
((2^19-1)/(2^19))^((2048-128)*1024) = 2.352...%
```
... probability that chunks end up reaching the maximum size? This sounds relatively high, but is arguably unavoidable.

A bit odd that these kinds of algorithms don't attempt to account for this, for example by repeatedly rerunning the algorithm with a smaller bit mask (18, 17, 16, [...] bits) until a match is found. That way you get a more even spread of data across such chunks, and need to upload fewer chunks in case data is injected into/removed from a sequence of chunks that would currently reach the 2 MB limit.